meson.build

```diff
@@ -157,7 +157,7 @@ flatc = find_program('flatc', required : get_option('flatbuf-support'))
 flatbuf_dep = disabler()
 flatbuf_version_check_dep = disabler()
 if flatc.found()
-  flatc_ver = run_command(flatc, '--version').stdout().split()[2]
+  flatc_ver = run_command(flatc, '--version', check : true).stdout().split()[2]
   flatbuf_dep = dependency('flatbuffers', version: flatc_ver,
```

Comment:
If you bump to Meson 0.62.0 (yes I know that's really new, you may not be able to), then you can use `flatc.version()`, see https://mesonbuild.com/Reference-manual_returned_external_program.html#external_programversion

Reply:
That's good; however, before applying this, we will wait (guess a year or two :( ) until we drop a few old versions of OS supported.

Reply:
Adding a TODO comment for the future is probably the best you can do for now then.

Reply:
I left a TODO comment for it. Thanks again.